Sources/HTTP/HTTPVersion.swift

```diff
@@ -46,12 +48,11 @@ extension HTTPVersion: Comparable {
             return lhs.minor < rhs.minor
         }
     }
-
 }
 
 extension HTTPVersion: CustomStringConvertible {
     /// :nodoc:
     public var description: String {
-        return "HTTP/" + major.description + "." + minor.description
+        return "HTTP/\(major.description).\(minor.description)"
```

Comment:
I don't know what is faster, I suspect `+`, but if you do `\()`, there is no need to call `description`. Do: `HTTP/\(major).\(minor)`.

Reply:
Oops yeah I missed that.